CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 - The `/eth/v1/beacon/blocks/:block_id` endpoint has been deprecated in favor of the v2 Altair endpoint `/eth/v2/beacon/blocks/:block_id`
 - The `/eth/v1/validator/blocks/:slot` endpoint has been deprecated in favor of the v2 Altair endpoint `/eth/v2/validator/blocks/:slot`
 - The `/eth/v1/debug/beacon/heads` endpoint has been deprecated in favor of the v2 Bellatrix endpoint `/eth/v2/debug/beacon/heads`
+- The `/eth/v1/node/peers` endpoint had field `address` in `data` object, which is not correct according to the spec, changed to `last_seen_p2p_address`. Also `meta` object with `count` added
+- The `/eth/v1/node/peers/{peer_id}` endpoint had field `address` in `data` object, which is not correct according to the spec, changed to `last_seen_p2p_address`
```

Comment:
This is the right content, but rather than `upcoming breaking change`, there should be a `breaking changes` section 3 sections lower.